help/C/Help_ch_Transactions.xml

```diff
@@ -56,14 +57,21 @@ Translators:
   <sect1 id="reg-views">
     <title>Changing the Register View</title>
 
-    <para><application>&app;</application> has several options to change the way the register looks so transactions can be seen more
-    clearly.  The default style is the Basic Ledger. This presents a single line view of transactions to allow
+    <para><application>&app;</application> has several options to change the way
+      the register looks so transactions can be seen more clearly.
+    </para>
+
+    <sect2 id="reg-views-view">
+      <title>View</title>
+    <para>
+    The default view is the Basic Ledger. This presents a single line view of transactions to allow
     the most concise view with all the relevant details.</para>
 
-    <para>The default view or style can be changed by going to <guimenu>View</guimenu> menu, three radio buttons
-    will select the desired option. When the window closes the style will revert to the Basic Ledger style. To
-    make this change permanent change the style in the <guilabel>Register Defaults</guilabel> tab of the
-    <guilabel>&app; Preferences</guilabel>.</para>
+    <para>The default view or style can be temporarily changed for the current
```

Comment:
Re View or Style:
I agree, consistently using View instead of Style or Mode would be better but the Preference is called "Default Style" so I didn't change what others have already written.
I think the preference should be renamed to "Default View".
Shall I raise a bug for that?
Shall I still remove Style from here?